README.md

```diff
@@ -1,2 +1,13 @@
-# faucet
-tendermint faucet
+# Faucet
+
+Use the following command to faucet tokens into your wallet address:
+
+```c
+curl -d '{"address":"cosmos1...<cosmos wallet address>"}' -H 'Content-Type: application/json' https://api.testnet.assetmantle.one/faucet/faucetRequest
```

Comment:
http://localhost:5000 instead of https://api.testnet.assetmantle.one/faucet/faucetRequest